test/multiverse/suites/stripe/stripe_instrumentation_test.rb

```diff
@@ -0,0 +1,160 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'json'
+require 'stripe'
+require 'net/http'
+
+class StripeInstrumentation < Minitest::Test
+  API_KEY = '123456789'
+
+  def setup
+    Stripe.api_key = API_KEY
+    # Creating a new connection and response, which both get stubbed
+    # later, helps us get around needing to provide a valid API key
+    @connection = Stripe::ConnectionManager.new
+    @response = Net::HTTPResponse.new('1.1', '200', 'OK')
+    # Bypass #stream_check ("attempt to read body out of block")
+    @response.instance_variable_set(:@read, true)
+    @response.body = {
+      object: 'list',
+      data: [{'id': '12134'}],
+      has_more: false,
+      url: '/v1/charges'
```

Comment:
It doesn't _really_ matter given that the tests don't operate on the url contained in the response, but from a "the specs should tell the story" perspective, perhaps you should store `'v1/customers'` as a constant and use it both here and in the `#detect` calls that the tests perform.

Reply:
updated https://github.com/newrelic/newrelic-ruby-agent/pull/2180/commits/63014dd5d7236c3fb38390840d7cfc0ff0ed70a4